### PR TITLE
Feat/password reset

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

File: app/lib/controllers/session.js [hunk 1]
```diff
@@ -55,26 +55,22 @@ Session.prototype.get = function (req, res, next) {
...
+  res.header('Content-Type', 'application/json')
+
   passport.authenticate('local', (err, user, info) => {
     if (err) {
       switch (err) {
         case 'MISSING_AUTH_API':
           res.statusCode = 503
           res.write(JSON.stringify(err))
 
-          break
```
Comment (eduardoboucas) [inline app/lib/controllers/session.js]: Why have you removed these?

Comment (mingard) [inline app/lib/controllers/session.js]: Mistake whilst refactoring

**Comment by eduardoboucas** [comment]
@mingard I have reinstated the `View` component. Shall we merge?

**Comment by mingard** [comment]
@eduardoboucas yep! Please merge and I'll test shortly

**Pull request merged**